helm/cluster-api/files/bootstrap/patches/versions/v1alpha2/kubeadmconfigs.bootstrap.cluster.x-k8s.io.yaml

```diff
@@ -327,16 +321,18 @@
                       description: BindPort sets the secure port for the API Server to bind to. Defaults to 6443.
                       format: int32
                       type: integer
-                  required:
-                    - advertiseAddress
-                    - bindPort
                   type: object
                 nodeRegistration:
                   description: NodeRegistration holds fields that relate to registering the new control-plane node to the cluster. When used in the context of control plane nodes, NodeRegistration should remain consistent across both InitConfiguration and JoinConfiguration
                   properties:
                     criSocket:
                       description: CRISocket is used to retrieve container runtime info. This information will be annotated to the Node API object, for later re-use
                       type: string
+                    ignorePreflightErrors:
```

Comment:
Not sure why we are getting v1alpha2 types updated here. Maybe it got added at some point for some compatibility reasons.

Reply:
mmm... this change was added in the new bootstrap CRDs version [that went from `v1beta1` to `v1beta2`](https://github.com/kubernetes-sigs/cluster-api/blob/4acc91a59250d5ae59ba177d4d8f6c91f65b2281/bootstrap/kubeadm/types/v1beta2/types.go#L214). But I'm confused now, since this is on the `v1alpha2` folder on our repo?

Reply:
It is coming from here from our fork, here it is in v1alpha2 https://github.com/giantswarm/cluster-api/blob/v1.0.0-alpha.1/bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml#L484

Reply:
That is `release-1.0` branch in our fork, which should be upstream `release-1.0` fork with `v1alpha2` added back.

Reply:
AFAIK we are not using bootstrap types from v1alpha2, so this should not be a problem.

Reply:
Only Kubeadm bootstrap types have these weird modifications, but we are not using v1alpha2 version of those types, so not an issue here.